src/catalogue/category/catalogueCardView.component.tsx

```diff
@@ -51,25 +51,19 @@ function CatalogueCardView(props: CatalogueCardViewProps) {
     endIndex
   );
 
+  const cardViewHeight = getPageHeightCalc('0px');
```

Comment:
If this is 0 then it presumably isn't needed. It isn't quite fixed though, while it doesnt create a horizontal scroll bar it isn't properly aligned. Should it be in a grid with two items with xs=6?
![image](https://github.com/ral-facilities/inventory-management-system/assets/90245114/8dd3a63e-6175-45a2-ae41-e66d7fe9ff40)